version.go

```diff
@@ -2,5 +2,5 @@ package awstpr
 
 const (
 	// VersionV1 is the Kubernetes API version for v1 cluster resources.
-	VersionV1 = "aws.cluster.giantswarm.io/v1"
+	VersionV1 = "cluster.giantswarm.io/v1"
```

Comment:
Why is that? Is there some specification? I thought it should be aligned to the TPR name.